### ISSUE TITLE
Not all rows shown in list

### ISSUE THREAD

**Issue by ashmigelski** [Polymer/core-list#45]
Hi

After initialization only first 8 rows shown, looks like on a screenshot. But if resize window (or click button in Demo) after loading - everything becomes ok
![image](https://cloud.githubusercontent.com/assets/4904519/5014150/776ff254-6aa4-11e4-8501-2fbae2f3e982.png)

Demo http://jsbin.com/lapimo/edit?html,output
My browser Chrome 38.0.2125.111

**Comment by zdarovka** [comment]
I think I have the same problem:

![core-list](https://lh3.googleusercontent.com/-MGVDFalB5us/VQATKR-QQSI/AAAAAAAARM0/qBy8DkAeGHE/w1358-h641-no/list.gif)

I have my st-pageable-content elements that extends the core-list

**Comment by dominik0711** [comment]
@zdarovka Could you solve the `core-list` bug somehow? My list behavior is looks the same and the hidden list items only show up when the browser window is resized. Any hints on that issue?

**Comment by zdarovka** [comment]
@dominik0711 Nope, I am no longer interested in polymer and the project that I was working on went into trash.

**Comment by dominik0711** [comment]
@zdarovka Do you use any alternatives for you web projects? AngularJS? MeteorJS?

**Comment by zdarovka** [comment]
@dominik0711 Nope, I am backend developer and this was my only frontend thing
